modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -25,27 +25,20 @@ The properties for broker-wide throughput quota balancing are configured at the
 | xref:reference:cluster-properties.adoc#kafka_throughput_limit_node_out_bps[kafka_throughput_limit_node_out_bps]
 | A broker's total throughput limit for egress Kafka traffic.
 
-| xref:reference:cluster-properties.adoc#kafka_quota_balancer_node_period_ms[kafka_quota_balancer_node_period_ms]
-| The period at which the quota balancer runs to balance throughput quota between a broker's shards.
-
-| xref:reference:cluster-properties.adoc#kafka_quota_balancer_min_shard_throughput_ratio[kafka_quota_balancer_min_shard_throughput_ratio]
-| The lowest value of the throughput quota a shard can get in the process of quota balancing, expressed as a ratio of the default shard quota. If set as `0`, there is no minimum, and if set as `1`, no quota can be taken away by the balancer.
+| kafka_throughput_control
+| List of clients for whom broker-wide limits do not apply
```

Comment:
Is this specifically worded differently than in the code?

> List of throughput control groups that define exclusions from node-wide throughput limits.

I ask because the configuration allows regex, so it doesn't have to be an exact match, which is somewhat implied by the wording above, but not so much by the text in the code.

Reply:
I'll have a commit in shortly on these two properties. Sorry for the confusion there. I'm mirroring what the code has and providing examples.